wrappers/wine-msvc.sh

```diff
@@ -15,49 +15,46 @@
 # OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 MSVCTRICKS_EXE="$(dirname $0)/../msvctricks.exe"
-EXE="$1"
+EXE=$1
 shift
 
 ARGS=()
-while [ $# -gt 0 ]; do
-	a=$1
-	case $a in
+for a; do
+	path=
+	case "$a" in
 	[-/][A-Za-z]/*)
-		opt=${a:0:2}
-		path=${a:2}
+		path=${a#??}
```

Comment:
any benefit using this form instead of the original one that substring by pos:len ?

Reply:
Broader compatibility with shells (POSIX standard), otherwise no